test/integration/test_raw_api.py

```diff
@@ -550,6 +551,71 @@ def raw_api_test_helper(raw_api, should_cleanup_old_buckets):
             is_file_lock_enabled=False,
         )
 
+    # b2_100_continue
```

Comment:
I don't like the way this test is done (multiple test cases in a single test, and mocking in Integration tests...), but I guess you inherited it ;p
Lets just make sure it working correctly.